data/ka'het/ka'het ships.txt

```diff
@@ -13,6 +13,7 @@ ship "Maeri'het"
 	sprite "ship/maeri'het"
 	thumbnail "thumbnail/maeri'het"
 	"never disabled"
+	"cookable"
```

Comment:
I like the keyword `cookable`, but I also feel that a configurable factor is more appropriate to allow some creatures to die at a factor 1.2 (like is done in this PR), while other creatures could die at a factor of 3. (something like `"fatal heat factor" 1.2` and `"fatal heat factor" 3.``.